src/engraving/style/style.cpp

```diff
@@ -113,6 +113,34 @@ int MStyle::defaultStyleVersion() const
     return styleI(Sid::defaultsVersion);
 }
 
+void MStyle::setStylePreset(ScoreStylePreset preset)
+{
+    m_stylepreset = preset;
+}
+
+void MStyle::setStylePresetEdited(bool isEdited)
+{
+    m_stylepresetedited = isEdited;
+}
+
+String MStyle::stylePreset2Name(ScoreStylePreset preset) const
```

Comment:
This function is no longer needed. You can use untranslated the TConv version instead.